survey-runner-routing/vpc_endpoint.tf

```diff
@@ -0,0 +1,11 @@
+resource "aws_vpc_endpoint" "private-dynamodb" {
```

Comment:
All of our other resources use `_` rather than `-`, so this should be `private_dynamodb`